packages/ndla-ui/src/ToggleLicenseBox/component.licensebox.scss

```diff
@@ -34,20 +77,15 @@
 }
 
 .c-licensebox--expanded {
-  background: $brand-color--lighter;
-  padding-top: $spacing;
-  z-index: 10;
-  color: $black;
-  box-shadow: 0 0 30px rgba($black, .2);
-  div {
-    animation-name: fadeInLeft;
-    animation-duration: .5s;
-  }
-  @include mq($from: desktop) {
-    @include grids-expand(4,6,2);
-    position: absolute !important;
+  display: block;
+
+  .c-licensebox__content {
```

Comment:
Tror du må ha en z-index her. Eventuelt et annet sted. Se «Bruk artikkel» i artikkelmal eksemplene. 

Itillegg ser heller ikke lisensbokssiden under sammensatte komponenter helt bra ut lenger.

Reply:
ja, har sett at det mangler en z-index. fikser det. har bare jobbet med den siden som er ekspanderbar. ser egentlig ikke nytten med å ha to sider. kanskje jeg skal fjerne denne siden og heller ha den ekspandert som default.

Reply:
Jepp, bare å fjerne den.